config/enterprise_versions.yml

```diff
@@ -95,7 +95,7 @@ components:
   # quay.io/tigera so all enterprise images come from the same repository and org.
   elasticsearch-operator:
     image: tigera/eck-operator
-    version: 1.3.2
+    version: 1.7.1
   prometheus:
     image: tigera/prometheus
```

Comment:
- [ ] note to move prometheus CRDs here as well

Reply:
https://github.com/tigera/operator/pull/1511/commits/14915a58f440a7e55058616da8a705c089a52b3f